zmon-controller-ui/js/controllers/AlertDefinitionEditCtrl.js

```diff
@@ -493,6 +493,16 @@ angular.module('zmon2App').controller('AlertDefinitionEditCtrl', ['$scope', '$ro
             return false;
         };
 
+        // Used by ui-select in view to return list of teams for Team field dropdown
+        // and allow inserting new values
+        $scope.getItems = function(prop, search) {
+            var options = prop ? [ prop ] : [];
+            if (search && options.indexOf(search) === -1) {
+                options.unshift(search);
```

Comment:
won't this add all my typed text to the options, i.e. every new character will add a new option?